lib/oli_web/components/delivery/scored_activities/scored_activities.ex

```diff
@@ -535,6 +535,9 @@ defmodule OliWeb.Components.Delivery.ScoredActivities do
         where: res_access.section_id == ^section.id,
         join: rev in Revision,
         on: aa.revision_id == rev.id,
+        left_join: rev2 in Revision,
+        on: rev.resource_id == rev2.resource_id and rev2.id > rev.id,
```

Comment:
This isn't correct - this will join to retrieve the latest version of this resource, which is not guaranteed to be the correct revision.  I think this query needs to group by `rev.resource_id`.  That will prevent duplicates due to differing revisions.  Upon selection of a row (i.e., an activity) this code should resolve the current version of that activity resource for that activity (regardless of how many revisions may have existed during the collection of the attempts)